tests/ut_seattlelib_time_interface_updatetime_before_registermethod.r2py

```diff
@@ -12,3 +12,6 @@ try:
 except time_interface.TimeError:
   # That's what we expect
   pass
+else:
+  # we have to fail this if we try didnt raise a exception for not registering time method
+  log("[FAIL]: Did not raise exception for not registering time method (ntp/tcp) \n")
```

Comment:
I think it's clearer if you include in the message the exception that should have been raised.
Suggestion: "time_interface.time_updatetime() succeeded but should have raised time_interface.TimeError"